### ISSUE TITLE
Excel like column filters?

### ISSUE THREAD

**Issue by stefaneidelloth** [adamerose/PandasGUI#180]
Are there plans to implement column filters like in Excel or qgrid?

https://github.com/quantopian/qgrid/blob/master/docs/images/filtering_demo.gif

![image](https://user-images.githubusercontent.com/12675339/147218723-c95c5c44-6c93-43a7-8995-ca01b6308317.png)

I would like to allow a user to filter a data frame in PandasGUI and afterwards
continue data processing on the filtered data frame.

**Comment by adamerose** [comment]
Yes there are definitely plans for this. I've been putting off work on features like this though because I'm considering moving from PyQt5 to React in 2022 and don't want to duplicate effort
